src/Core/Domain/Alias/Exception/AliasNotFoundException.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Domain\Alias\Exception;
+
+use PrestaShop\PrestaShop\Core\Domain\Alias\ValueObject\AliasId;
+
+class AliasNotFoundException extends AliasException
+{
+    /**
+     * @var AliasId
+     */
+    private $aliasId;
+
+    /**
+     * @param AliasId $aliasId
+     * @param $message
+     * @param $code
+     * @param $previous
+     */
+    public function __construct(AliasId $aliasId, $message = '', $code = 0, $previous = null)
```

Comment:
```suggestion
```
don't need an id in constructor. (there are only couple domains that uses it and we consider them mistakes)